anta/tests/routing/isis.py

```diff
@@ -602,129 +617,36 @@ def test(self) -> None:
         for input_entry in self.inputs.entries:
             eos_entry = self._eos_entry_lookup(search_value=input_entry.endpoint, entries=command_output["entries"])
             if eos_entry is None:
-                failure_message.append(f"Tunnel to {input_entry} is not found.")
+                failure_message.append(f"Tunnel to {input_entry.endpoint!s} is not found.")
```

Comment:
For this condition you could use `get_item` from our tools module and remove `_eos_entry_lookup` method.

Reply:
Recently we started to use `self.result.is_failure` directly to have a list of failure messages in `TestResult`.
```suggestion
                self.result.is_failure(f"Tunnel to {input_entry.endpoint!s} is not found.")
```

Reply:
You wouldn't need `failure_message` in that case.